wormhole-connect/package.json

```diff
@@ -129,7 +129,7 @@
     "react-dom": "^18.2.0"
   },
   "lint-staged": {
-    "**/*": "prettier --workspaces --if-present --write --ignore-unknown"
```

Comment:
we also do not need these two params, workspaces is no longer and if-present is deprecated.